SuperBuild/External_ffi.cmake

```diff
@@ -0,0 +1,86 @@
+
+set(proj ffi)
```

Comment:
for consistency, I will likely rename the project to `libffi`. See http://sourceware.org/libffi/

Reply:
Will also probably renamed the variables in `python-cmake-buildsystem` to have the prefix `LIBFFI_`